lipnet/lipreading/videos.py

```diff
@@ -191,11 +192,30 @@ def get_frames_mouth(self, detector, predictor, frames):
             mouth_frames.append(mouth_crop_image)
         return mouth_frames
 
-    def get_video_frames(self, path):
+    def get_video_frames_skvideo(self, path):
```

Comment:
Don't change the current method of video loading as it has been proven to be reliable and doesn't require an additional installation of OpenCV, which is sometimes problematic (not to mention that the program itself sometimes doesn't work as expected).

Instead, provide an additional argument for the user in the ``from_video`` method. Something like this: ``from_video(self, path, use_open_cv=False)``. Please keep Sk-video to be the default method.

Reply:
Done!